src/parser/mod.rs

```diff
@@ -88,29 +88,59 @@ fn test_parse_value_data_2() -> Result<(), Error> {
     Ok(())
 }
 
+/// Parse a scss file by path.
+///
+/// Returns a vec of the top level items of the file (or an error message).
+pub fn parse_scss_path(path: &Path) -> Result<Vec<Item>, Error> {
+    do_parse_scss_path(path, &SourceName::root(path.display()))
+}
+
+pub fn parse_imported_scss_path(
```

Comment:
I think this function (`parse_imported_scss_path`) should be removed.  The `_imported_` variant is only intended for internal use as rsass handles an `@import` statement, and this PR changes that to not use the `_path` but the `_file` variant.  So this function is not needed.

Reply:
I agree